cli/Cargo.toml

```diff
@@ -42,6 +42,7 @@ dlopen = "0.1.8"
 encoding_rs = "0.8.23"
 dprint-plugin-typescript = "0.30.1"
 futures = "0.3.5"
+filetime = "0.2.12"
```

Comment:
Is this necessary?

Reply:
We can do it by libc and winapi I suppose but I'm satisfied with the implementation (uses futimens) in filetime and it makes it a readable one liner.

Reply:
Oh and the utime crate we currently have a dependency on doesn't provide any fd/handle based functions, just the one path based one.

So to clarify; the choices are filetime crate or write libc code duplicating what the filetime crate does.

Reply:
Fair enough. I will land this as is then.